actors/builtin/multisig/multisig_test.go

```diff
@@ -89,7 +89,7 @@ func TestVesting(t *testing.T) {
 
 	const unlockDuration = 10
 	var multisigInitialBalance = abi.NewTokenAmount(100)
-	var nilParams = runtime.CBORBytes([]byte{})
```

Comment:
removed all references to `nilParams` and use `fakeParams` everywhere as sending actual content is more robust for testing.